src/Plotting.jl

```diff
@@ -253,6 +253,29 @@ function plot_spins(sys::System; linecolor=:grey, arrowcolor=:red,
         linecolor, arrowcolor, linewidth, arrowsize,
         lengthscale=arrowlength, kwargs...
     )
+
+    if !isnothing(sys.origin)
+        sys_origin = resize_periodically(sys,sys.origin.latsize)
+
+        # Translate to a correct lattice site in the middle of the bigger system
+        center_origin = (sys_origin.crystal.latvecs * Vec3(sys_origin.latsize))/2
+        center_this = (sys.crystal.latvecs * Vec3(sys.latsize))/2
+        center_offset = center_origin .- center_this
+        middle_site = global_position(sys_origin,CartesianIndex((sys_origin.latsize .÷ 2)...,1))
+        diff_center = center_offset .- middle_site
+
+        pts = Makie.Point3f0.(map(x -> diff_center .+ x,spin_vector_origins(sys_origin, arrowlength))[:])
+        vecs = Makie.Vec3f0.(sys_origin.dipoles[:])
+        Makie.arrows!(
+            ax, pts, vecs;
+            linewidth, arrowsize,
+            lengthscale=arrowlength,
+            # Original system is transparent
+            arrowcolor=(arrowcolor,0.03), linecolor = (linecolor,0.03),
+            kwargs...
+        )
+    end
```

Comment:
This may have been better as a separate PR, but it does this:

![image](https://github.com/SunnySuite/Sunny.jl/assets/2996388/0b64fc4b-70f4-4d70-892b-21b4b029577a)

which I think is neat and resolves the confusion I had looking at this example for the longest time (I thought "supercell" meant that each of the four spins was meant to represent a cluster of spins in the original system, rather than a sublattice)